bpf/flows.c

```diff
@@ -97,7 +138,7 @@ static inline int fill_iphdr(struct iphdr *ip, void *data_end, flow_id *id) {
 }
 
 // sets flow fields from IPv6 header information
-static inline int fill_ip6hdr(struct ipv6hdr *ip, void *data_end, flow_id *id) {
+static inline int fill_ip6hdr(struct ipv6hdr *ip, void *data_end, flow_id *id, u16 *flags) {
```

Comment:
shouldn't we call `set_flags`  here too, in `case IPPROTO_TCP` block?

Reply:
Good catch! We always forget to test IPv6 :-(